<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffOscWgtClass</class>
 <widget class="QWidget" name="EffOscWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EffOscWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_WndSpecAns">
      <attribute name="title">
       <string>窗谱分析</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_baseSig_chart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>800</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1980</width>
             <height>440</height>
            </size>
           </property>
           <property name="title">
            <string>调制窗函数绘制</string>
           </property>
           <layout class="QHBoxLayout" name="HLayout_TimeCht_Wnd"/>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>STFT窗参数</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>窗型</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_WndType">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>矩形</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Hann</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>长度定义</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QRadioButton" name="radioBtn_WndLengthType_Point">
                  <property name="text">
                   <string>点数</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioBtn_WndLengthType_Time">
                  <property name="text">
                   <string>时长</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_WndLengthPoint">
                 <property name="text">
                  <string>窗型点数：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="SBox_WndLengthPointCount">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="value">
                  <number>4096</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_WndSampleRate">
                 <property name="text">
                  <string>采样率</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="SBox_WndSampleRate">
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="value">
                  <number>44100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_WndLength_Time">
                 <property name="text">
                  <string>窗型时长(ms)：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="DSBox_WndLength_TimeMSec">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <double>100000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>100.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_resultWndLength_Point">
               <property name="text">
                <string>窗型等价点数：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_resultWndLength_Time">
               <property name="text">
                <string>窗型等价时长：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_WndSTFTFreqDelta">
               <property name="text">
                <string>FFT输出频率间隔：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_GenWndTimeGraph">
               <property name="text">
                <string>生成窗型</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>冲激调制参数</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_11">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>408</height>
                </size>
               </property>
               <property name="title">
                <string>窗函数参数</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLabel" name="label_ActiveWnd_sampleRate">
                  <property name="text">
                   <string>采样率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_ActiveWnd_WndType">
                  <property name="text">
                   <string>窗函数类型</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_ActiveWnd_FFTWndPtCount">
                  <property name="text">
                   <string>窗函数点数</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <item>
                <widget class="QLabel" name="label_ActiveStartRatio">
                 <property name="text">
                  <string>开始位置比例（%）</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="DSBox_ActiveStartRatio">
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>34.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QLabel" name="label_IncreaseTime">
                 <property name="text">
                  <string>上升时间（ms）</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="DSBox_IncreaseTime">
                 <property name="maximum">
                  <double>50.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>15.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btn_GenActiveWndTimeGraph">
               <property name="text">
                <string>生成调制函数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_ActiveWnd_ShowType">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>显示对象</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_34">
                <item>
                 <widget class="QCheckBox" name="checkBox_ActiveWnd_ShowType_STFTWnd">
                  <property name="text">
                   <string>STFT窗</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_ActiveWnd_ShowType_ActiveVal">
                  <property name="text">
                   <string>激发函数</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_ActiveWnd_ShowType_ActiveWnd">
                  <property name="text">
                   <string>激发窗</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_3">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_activeWndSpec">
          <attribute name="title">
           <string>激发调制窗频谱</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <widget class="QGroupBox" name="groupBox_15">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="title">
              <string>窗形频谱</string>
             </property>
             <layout class="QVBoxLayout" name="VLayout_ActiveFFTWndSpectrum_Cht"/>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_12">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>分析控制</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_13">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="title">
                 <string>值显示方式</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_26">
                 <item>
                  <widget class="QRadioButton" name="radioButton_ActiveWndFFTValType_AbsoluteVal">
                   <property name="text">
                    <string>绝对值</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_ActiveWndFFTValType_db">
                   <property name="text">
                    <string>db</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_ActiveWndFFTValType_AbsDb">
                   <property name="text">
                    <string>绝对db</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_27">
                <item>
                 <widget class="QLabel" name="label_ActiveWndFFTShowPointCount">
                  <property name="text">
                   <string>总点数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="SBox_ActiveWndFFTShowPointCount">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>2000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_28">
                <item>
                 <widget class="QLabel" name="label_ActiveWndFFTShowMaxFreq">
                  <property name="text">
                   <string>显示最大频率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="SBox_ActiveWndFFTShowMaxFreq">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                  <property name="value">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_29">
                <item>
                 <widget class="QLabel" name="label_ActiveWndFFTShowMinVal">
                  <property name="text">
                   <string>显示最小数值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="SBox_ActiveWndFFTShowMinVal">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimum">
                   <number>-10000</number>
                  </property>
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                  <property name="value">
                   <number>-100</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_ActvieWndFFTGenState">
                <property name="text">
                 <string>生成状态</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="progressBar_ActvieWndFFTGenState">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="btn_GenActiveWndFFTGraph">
                  <property name="text">
                   <string>频谱分析</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_updateActiveWndFFTCht">
                  <property name="text">
                   <string>刷新图像</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>窗谱抑制输出</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
